GitExtensions/GitExtensions.csproj

```diff
@@ -10,7 +10,7 @@
     <ApplicationIcon>..\Logo\git-extensions-logo.ico</ApplicationIcon>
     <StartupObject />
 
-    <Prefer32bit>true</Prefer32bit>
+    <!-- Prefer32bit>true</Prefer32bit -->
```

Comment:
I'm not sure we can do this; this change will require further investigation. See https://github.com/gitextensions/gitextensions/issues/9990

Reply:
I have no problem with that setup with GE in both x64 and WoA platforms. I haven't seen the commit problem mentioned in that issue either.

Reply:
Let's keep this enabled. This can be disabled as `/p:Prefer32bit=false` during the ARM64 builds.

Reply:
OK, although leaving it enabled makes no difference in the resulting GE software for WoA, as far as I can determine, so the `/p:Prefer32bit=false` may not be needed for WoA build.